packages/exhibitor-migration-script/extra/exhibitor-migration-script/bin/dcos-exhibitor-migrate-perform

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import copy
+
+import requests
+import sys
+from requests import RequestException
+
+ZOOKEEPER_DATA_DIR_START = '/var/lib/zookeeper/snapshot'
+ZOOKEEPER_LOG_DIR_START = '/var/lib/zookeeper/transactions'
+LOG_INDEX_DIR_START = '/var/lib/zookeeper/transactions'
+
+ZOOKEEPER_DATA_DIR_GOAL = '/var/lib/dcos/exhibitor/zookeeper/snapshot'
+ZOOKEEPER_LOG_DIR_GOAL = '/var/lib/dcos/exhibitor/zookeeper/transactions'
+LOG_INDEX_DIR_GOAL = '/var/lib/dcos/exhibitor/zookeeper/transactions'
+
+
+class ValidationError(Exception):
+    def __init__(self, msg, exit_status=8):
+        self.msg = msg
+        self.exit_status = exit_status
+
+
+def get_config_json(response):
+    if response.status_code == 200 and response.headers['Content-Type'] == 'application/json':
+        return response.json()['config']
```

Comment:
might be worthwhile to check the json object is non-empty (has a config key) based on previous errors we've seen in the dc/os integration tests.